app/views/games/show.html.erb

```diff
@@ -1,9 +1,24 @@
 <h1 class="text-center">Have Fun</h1>
 
+<!-- drawing chess board -->
 <table class='chess-board chess-squares'>
-  <% @rows.times do |row| %>
+  <% 8.times do |row| %>
     <tr>
-      <% @columns.times do |column| %><td id="<%= (row + 1) * (column + 1) %>"></td><% end %>
+      <% 8.times do |column| %><td id="<%= (row * 8) + (column + 1) %>"></td><% end %>
     </tr>
   <% end %>
 </table>
+
+<% @game.pieces.each do |piece| %>
```

Comment:
Couple of small things here:

* This is going to generate a lot of different script tags side-by-side, one for each piece
* This code gets executed as soon as the page gets to it.  Not on page load.  We should wrap this in a jQuery on load event.  i.e. code that looks like:

```
$(function() {
 // ALL JS HERE
});
```

Instead of doing the work in a mix of ruby/javascript I might suggest to keep the mixing of the languages to a minimum.  Think about using code like this:

```
var pieces = <%= @game.pieces.to_json %>;
```

Then you can do it all in JavaScript.  On the flip side, I feel like this could be doable without JS.